src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started to work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped to work");

```

Comment:
Similarly to the doWork method, the message "Truck stopped to work" should be more informative. Indicate that it is a Truck that has stopped working and specify the action if possible.